dash_app/views/worldmap.py

```diff
@@ -41,13 +40,17 @@ def world_map(request):
             'x':0.5,
             'xanchor': 'center',
             'yanchor': 'top'
-        })
+        },
+        geo=dict(
+            showframe=False,
+            showcoastlines=False
+        )
+    )
 
-    fig.update_geos(showcountries=True, countrywidth=0.2, countrycolor="Black",
-                    lataxis_range=[-60, 90])
+    fig.update_geos(showcountries=True, countrywidth=0.2, lataxis_range=[-60, 90])
```

Comment:
Why are these geo settings segregated?

I think it should be possible to define **showcountries**, **countrywidth** and **lataxis_range** inside the geo dictionary.

Reply:
There's no specific reason, sounds like a good idea to combine the settings into a single place.

Reply:
This was applied in commit [37026da](https://github.com/elixir-europe-training/ELIXIR-TrP-Training-Metrics-Database-Tango/pull/23/commits/37026da28c7a7581c8efab6747d9f094de9fce2a) so I hope it's okay that I resolve this conversation.